src/validation/coinbase.cairo

```diff
@@ -30,7 +30,12 @@ pub fn validate_coinbase(
     let block_reward = compute_block_reward(block_height);
     assert(total_output_amount <= total_fees + block_reward, 'total output > block rwd + fees');
 
-    // TODO: validate BIP-34 sig script
+    // validate BIP-34 sig script
+    if block_height >= 227836 {
+        let sig = tx.inputs[0].script.clone();
```

Comment:
It makes sense to declare `sig_len = tx.inputs[0].script.len()` use use it below instead of calling `len` twice..